### ISSUE TITLE
Entire site sometimes turns blank when editing rich text

### ISSUE THREAD

**Issue by AlessioGr** [payloadcms/payload#568]
# Bug Report
This just happened to me while editing rich text (I just typed a letter):
![image](https://user-images.githubusercontent.com/70709113/169101006-e1bd3eab-edb8-4afb-80be-94c2f1ced89a.png)
![image](https://user-images.githubusercontent.com/70709113/169101087-b4e7f3d5-3d4c-4bbc-b4af-e0b542ca9276.png)

I don't know why, it happens seemingly randomly. It's quite annoying as I have to reload the page to fix it - losing all unsaved progress.

I cannot reproduce it, but maybe the console error already helps you. I'll add to this issue here when I can reliably reproduce it (or maybe others are having the same issue)

**Comment by jmikrut** [comment]
Hey @AlessioGr — we've seen this once or twice as well and typically it has to do with either a custom SlateJS element or leaf, OR it could have to do with a specific data structure that is currently saved within your document.

Next time this happens, would you mind sharing exactly what you were doing before it happened? Specifically, what element or leaf you were using?

**Comment by Tapiwachin** [comment]
For me whenever i try to use that HR in the text editor it shows a blank screen and some slate errors in the console... 
![Capture-2](https://user-images.githubusercontent.com/1384308/182109287-7213c9d7-b95b-447b-bb47-5cb58d5eab6d.PNG)

**Comment by jmikrut** [comment]
@Tapiwachin — really? I can't reproduce that. Can you reproduce with the Payload demo?

https://demo.payloadcms.com/

**Comment by Tapiwachin** [comment]
> @Tapiwachin — really? I can't reproduce that. Can you reproduce with the Payload demo?
> 
> https://demo.payloadcms.com/

No, demo seems to be fine.  I repurposed collections from here https://medium.com/trbl/building-a-professionally-designed-website-episode-4-heroes-layout-building-blocks-animations-6e819b1a5bba onto the latest payload version. Maybe there is something that i have missed.

**Comment by AlessioGr** [comment]
> Hey @AlessioGr — we've seen this once or twice as well and typically it has to do with either a custom SlateJS element or leaf, OR it could have to do with a specific data structure that is currently saved within your document.
> 
> Next time this happens, would you mind sharing exactly what you were doing before it happened? Specifically, what element or leaf you were using?

It happened again! I've recorded it here: https://youtu.be/VfD3iK5Cl2I

**Comment by jmikrut** [comment]
Hey @AlessioGr — thank you very much for the video! I was able to reproduce. I had to go to the URL in your video and copy / paste a similar piece of content as you to reproduce, but that led me to a few improvements that we could make to the rich text editor copy / paste HTML deserializing.

This will be released in our next patch version, possibly later tonight but definitely within the next 24hr or so!
